### PR TITLE
Add support for non-sortable full-width postboxes

### PR DESCRIPTION
**Pull request by l3ku**
#### What are the main changes in this PR?
Add support for displaying full-width postboxes on the top of `seravo_postboxes_page` by adding the `'full_width'` postbox content parameter that can be used as the fifth parameter for `seravo_add_postbox()`.

##### Why are we doing this? Any context or related work?
Related to the discussions in https://github.com/Seravo/seravo-plugin/issues/233.

#### Where should a reviewer start?
Locally or on a test site, try adding postboxes to pages with `full_width` context, e.g.:
```
seravo_add_postbox(
   'full-width-test',
   __('Full-width Postbox', 'seravo'),
   array( __CLASS__, 'database_adminer_postbox' ),
   'tools_page_database_page',
   'full_width'
);
```

#### Manual testing steps?
The full-width postboxes should be displayed on the top of the page before the sortable postboxes.

#### Screenshots
Firefox_Screenshot_2019-08-30T08-21-32 708Z
Firefox_Screenshot_2019-08-30T07-46-09 691Z
Firefox_Screenshot_2019-08-30T07-29-11 729Z

### CODE REVIEW

**Comment by l3ku** [comment]
> Overally LGTM. But there is one thing.

What thing? I also can't see any code comments which would request changes?

File: lib/seravo-postbox.php [hunk 1]
```diff
@@ -263,8 +272,15 @@ class="postbox seravo-postbox <?php echo $closed ? 'closed' : ''; ?>">
           <span class="toggle-indicator" aria-hidden="true"></span>
         </button>
 
+        <?php
+        // All other contexts but full_width are sortable
+        $handle_class = 'hndle';
+        if ( $postbox_content !== 'full_width' ) {
```
Comment (cr0vy) [inline lib/seravo-postbox.php]: Most likely typo?

Comment (l3ku) [reply lib/seravo-postbox.php]: Thanks for noticing, I have now fixed the variable name from `$postbox_content` to `$postbox_context`.

**Comment by l3ku** [comment]
@cr0vy what is your own opinion about what would be the best features into postboxes or other tools that would help in solving the UI challenges in https://github.com/Seravo/seravo-plugin/issues/233? Would you think that this this full-width postbox a good tool for that along with support for limiting the amount of postbox columns? What about the tabs I showcased in https://github.com/Seravo/seravo-plugin/issues/233#issuecomment-526590313, maybe we could also add support for non-postbox tabs?

**Comment by cr0vy** [comment]
@l3ku I like the idea of the tabs, but my opinion is that the best solution for the issue #233 is limiting amount of postbox columns. At the same time, we should consider that some pages could need full-width postboxes, too. 

Also, I tested full-width postboxes with toolbox code and full-width postboxes broke up on some pages. It looks like full-width postboxes don't like, if two or more separate php files (example: database.php and reports.php) try to add postboxes for the page.

**Comment by l3ku** [comment]
> @l3ku I like the idea of the tabs, but my opinion is that the best solution for the issue #233 is limiting amount of postbox columns. At the same time, we should consider that some pages could need full-width postboxes, too.
> 
> Also, I tested full-width postboxes with toolbox code and full-width postboxes broke up on some pages. It looks like full-width postboxes don't like, if two or more separate php files (example: database.php and reports.php) try to add postboxes for the page.

Nothing should break in full-width postboxes even if different PHP files register them, as long as they are registered to the same admin screen/page. If you find a better mechanism for creating full-width postboxes along with the column width limitations, feel free to implement them in a separate development branch.

**Comment by ottok** [comment]
What do you think, do we need this or should we scrap the idea (and thus implementation)?

**Comment by l3ku** [comment]
If there is no need for this, we might as well drop it.

**Pull request closed without merging**